<!-- {acknowledgement} --><a name="acknowledgement"></a>
<div class="info-box embedded-box"><p><a href="http://www.vectorbase.org/"><img alt="VectorBase" class="float-right" src="/img/vectorbase.jpg" style="height:38px; width:202px" /></a> The <em>Anopheles gambiae</em> data and its display on Ensembl Genomes are made possible through a joint effort by the Ensembl Genomes group and VectorBase, a <a href="http://www.pathogenportal.org">NIAID Bioinformatics Resource Center</a>. The source data for this genome and other resources can be accessed via the <a href="http://www.vectorbase.org/Anopheles_gambiae/Info/Index">VectorBase browser</a>.</p>
</div>
<!-- {acknowledgement} -->
<!-- {about} --><a name="about"></a>
<h2 id="about">About <em>Anopheles gambiae</em></h2>

<p><em>Anopheles gambiae senso stricto</em> is the primary mosquito vector responsible for the transmission of malaria in most of sub-Saharan Africa. It is a member of a species complex that includes at least seven morphologically indistinguishable species in the Series Pyretophorus in the Anopheles subgenus Cellia. <em>An. gambiae</em> feeds preferentially on humans and is one of the most efficient malaria vectors known.</p>

<p>Picture credit (public domain): <a href="http://phil.cdc.gov/phil/details_linked.asp?pid=1354">James Gathany</a> (CDC) 1994</p>

<p><!--nbsp--> <!--nbsp--></p>

<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<h2 id="assembly">Assembly</h2>

<p>The genome assembly presented here (AgamP4, April 2014) is a revised assembly based on the whole genome shotgun assembly of the PEST strain of <em>Anopheles gambiae</em> [1, 2], plus the addition of a mitochondrial chromosome. The assembly is described in detail in <a href="https://www.vectorbase.org/organisms/anopheles-gambiae/pest/agamp4">VectorBase</a>.</p>

<p><!--nbsp--> <!--nbsp--></p>

<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h2 id="annotation">Annotation</h2>

<p>Annotation of the AgamP4 assembly was carried out by VectorBase. The set of gene models presented (genebuild AgamP4.3, released October 2015) combines manual annotation, data provided by the research community, and gene prediction using the Ensembl system. Prediction utilised alignments of dipteran and other protein sets to the genome and generation of GeneWise models, alignment and gene prediction based on Anopheles ESTs, and selected <em>ab initio</em> predictions. More details can be found in <a href="https://www.vectorbase.org/organisms/anopheles-gambiae/pest/AgamP4.1">VectorBase</a>.</p>

<p><!--nbsp--> <!--nbsp--></p>

<h2>Functional Genomics</h2>

<p>The functional genomics database for <em>An. gambiae</em> contains mappings to probes from the many microarray designs, more details can be found at the <a href="http://funcgen.vectorbase.org/ExpressionData/microarray/">VectorBase microarray page</a>.</p>

<p><!--nbsp--> <!--nbsp--></p>

<!-- {annotation} -->
<!-- {variation} --><a name="variation"></a>
<h2 id="variation">Variation</h2>

<p>Variation data for <em>Anopheles gambiae</em> was imported from <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">NCBI dbSNP</a>, and from other studies involving the M and S molecular forms [3, 4, 5].</p>

<p>Variation data is also available for the <em>Anopheles gambiae</em> MR4 reference colonies 4ARR, Kisumu, Akron, L3-5 and G3. These samples were sequenced by the <a href="http://www.sanger.ac.uk/research/projects/malariaprogramme-kwiatkowski/#t_research">Kwiatkowski group at the Wellcome Trust Sanger Institute</a>, as part of the Malaria Programme's <em>Anopheles gambiae</em> Genome Variation Project. These variants should be considered preliminary, pending further analysis and quality control filtering.</p>

<p><!--nbsp--> <!--nbsp--></p>

<!-- {variation} -->
<!-- {other} --><a name="other"></a>
<h2 id="other">EST and Protein Alignments</h2>

<p><em>Anopheles gambiae</em> ESTs were mapped onto the genome using <a href="http://www.biomedcentral.com/1471-2105/6/31">Exonerate</a> (Example: <a href="http://metazoa.ensembl.org/Anopheles_gambiae/Share/a2cb3c3445bf3ae552dc1377820c215667219496">2L:155000-225000</a>).</p>

<p>Proteins from all VectorBase species were mapped to the <em>Anopheles gambiae</em> genome (using blastx) (Example: <a href="http://metazoa.ensembl.org/Anopheles_gambiae/Share/4db0ed23a5214d427b1580039328cf1b67219496">2L:39300000-39320000</a>).</p>

<!-- {other} -->
<h3>References</h3><ol><li><a id="ref-01"></a><a href="http://europepmc.org/abstract/MED/12364791">The genome sequence of the malaria mosquito <em>Anopheles gambiae</em></a>.<br>Holt RA, Subramanian GM, Halpern A, Sutton GG, Charlab R, Nusskern DR, Wincker P, Clark AG, Ribeiro JM, Wides R et al. 2002. Science. 298:129-149.</li>
<li><a id="ref-02"></a><a href="http://europepmc.org/abstract/MED/17210077">Update of the <em>Anopheles gambiae</em> PEST genome assembly</a>.<br>Sharakhova MV, Hammond MP, Lobo NF, Krzywinski J, Unger MF, Hillenmeyer ME, Bruggner RV, Birney E, Collins FH. 2007. Genome Biology. 8:R5.</li>
<li><a id="ref-03"></a><a href="http://europepmc.org/abstract/MED/20966254">SNP genotyping defines complex gene-flow boundaries among African malaria vector mosquitoes</a>.<br>Neafsey DE, Lawniczak MK, Park DJ, Redmond SN, Coulibaly MB, Traor SF, Sagnon N, Costantini C, Johnson C, Wiegand RC et al. 2010. Science. 330:514-517.</li>
<li><a id="ref-04"></a><a href="http://europepmc.org/abstract/MED/20976111">Association mapping of insecticide resistance in wild <em>Anopheles gambiae</em> populations: major variants identified in a low-linkage disequilbrium genome</a>.<br>Weetman D, Wilding CS, Steen K, Morgan JC, Simard F, Donnelly MJ. 2010. PLoS ONE. 5:e13140.</li>
<li><a id="ref-05"></a><a href="http://europepmc.org/abstract/MED/21836185">Gene flow-dependent genomic divergence between <em>Anopheles gambiae</em> M and S forms</a>.<br>Weetman D, Wilding CS, Steen K, Pinto J, Donnelly MJ. 2012. Molecular Biology and Evolution. 29:279-291.</li></ol>